src/platform/windows/gui/help/sysinfo.cpp

```diff
@@ -0,0 +1,98 @@
+// Reference : https://stackoverflow.com/questions/6121792/how-to-check-if-a-cpu-supports-the-sse3-instruction-set
+#if defined(_M_IX86) || defined(_M_X64)
+#include "sysinfo.h"
+#include <imgui.h>
+#include <intrin.h>
+#include <cstdint>
+
+namespace gui::help {
+
+void SysInfo::sysinfoWindow() {
+    ImGui::Begin("System Information", &sysinfoWindowOpen, ImGuiWindowFlags_AlwaysAutoResize);
+    ImGui::Text("%s", cpu_info.brand_string);
+    if (cpu_info.bLongMode)
+        ImGui::Text("Your CPU have 64-bit");
+    else
+        ImGui::Text("Your CPU have 32 bit");
+    ImGui::End();
+}
+
+void SysInfo::displayWindows() {
+    if (sysinfoWindowOpen) sysinfoWindow();
+}
+
+void do_cpuidex(uint32_t regs[4], uint32_t cpuid_leaf, uint32_t ecxval) { __cpuidex((int *)regs, cpuid_leaf, ecxval); }
+void do_cpuid(uint32_t regs[4], uint32_t cpuid_leaf) { __cpuid((int *)regs, cpuid_leaf); }
+
+CPUInfo cpu_info;
+
+CPUInfo::CPUInfo() { 
+	Detect(); 
+}
+
+// Detects the various cpu features
+void CPUInfo::Detect() { 
+	memset(this, 0, sizeof(*this));
```

Comment:
"memset" is a big no-no in this case. It's the constructor's responsibility to initialize member fields, but field initialization is often enough.